hw/BUILD

```diff
@@ -17,6 +17,37 @@ fusesoc_build(
     data = ["//hw/ip/otbn:all_files"],
     systems = ["lowrisc:dv:chip_verilator_sim"],
     target = "sim",
+    visibility = ["//visibility:private"],
+)
+
+# This is used in CI steps that do not want to run Verilator tests, and thus
+# do not want to accidentally build Verilator. This causes the Verilator
+# target to not emit any files, which will break any tests that actually rely
+# on this.
+config_setting(
+    name = "fake_fusesoc",
+    values = {"define": "DISABLE_VERILATOR_BUILD=true"},
+)
```

Comment:
I'm not really sure I understand the motivation here: if you don't want to run stuff that needs Verilator, can't you just not depend on it? This looks like the sort of cleverness that just confuses people down the line.

Reply:
Please create an issue to understand what is happening with the dependency graph with the goal of eliminating this `config_setting`.  Leave a TODO comment here referencing the issue.